packages/emulation/libretro-swanstation/package.mk

```diff
@@ -0,0 +1,40 @@
+# SPDX-License-Identifier: GPL-2.0
+# Copyright (C) 2016-present Team LibreELEC (https://libreelec.tv)
+
+PKG_NAME="libretro-swanstation"
+PKG_VERSION="6a8c32bb17df77e512a434f622ece1957f391ad8"
+PKG_SHA256="8ef44b88e057159c3b5fded06824494f4d3f5a5eea11e2e6f930ec42b15c3485"
+PKG_LICENSE="GPL-3.0-or-later"
+PKG_SITE="https://github.com/libretro/swanstation"
+PKG_URL="https://github.com/libretro/swanstation/archive/${PKG_VERSION}.tar.gz"
+PKG_DEPENDS_TARGET="toolchain"
+PKG_LONGDESC="SwanStation is a hard fork and open-source Libretro core implementation of DuckStation, which is an emulator of the Sony PlayStation(TM) console, focusing on playability, speed and long-term maintainability."
```

Comment:
I'd simplify this to "SwanStation is an open-source Libretro core implementation of DuckStation, an emulator of the Sony PlayStation console."

Reply:
And maybe the add-on language strings themselves can be changed.

Reply:
I'm not a native English speaker, but is the formulation "an open-source implementation of DuckStation" the same as "a hard fork of DuckStation"?

Reply:
Maybe I would only omit the "focusing on ..." part and keep the rest as it is, because it is copied from the official project web.

"SwanStation is a hard fork and a Libretro core implementation of DuckStation, which is an emulator of the Sony PlayStation console."

Like this?

Reply:
I've edited the descriptions of both packages to match the other emulators.

Reply:
I'm a native speaker and general distro nerd and I couldn't tell you what differentiates a "hard" code fork from any other kind of code fork. It's not language that mortal users comprehend or care about so I simplified.

Reply:
If I get it right, then the "hard" fork means that the code was copied over to a new project and the link to the original project was completely removed. The new project is then completely independent and goes its own way. The SwanStation repository is not marked as a fork of DuckStation:
https://github.com/libretro/swanstation